extensions/amp-story/0.1/amp-story.js

```diff
@@ -1282,7 +1302,7 @@ export class AmpStory extends AMP.BaseElement {
    * @return {number}
    */
   getPageCount() {
-    return this.pages_.length;
+    return this.pages_.length - this.adPages_.length;
```

Comment:
as Jon suggested, we can filter pages by tag name.

Reply:
same as above, will need to build all filtering into amp-story. I thought that would be a future PR but happy to include it here